concordium-consensus/tools/database-exporter/DatabaseExporter/CommandLineParser.hs

```diff
@@ -38,6 +39,18 @@ config =
                                             <> metavar "NUM"
                                             <> help "Maximum number of blocks to export in a single file"
                                         )
+                                    <*> option
+                                        auto
+                                        ( long "overwrite-chunks"
+                                            <> value False
+                                            <> showDefault
+                                            <> metavar "BOOL"
+                                            <> help
+                                                ( "Overwrite chunk files if they already exist; "
+                                                    <> "otherwise choose an available filename by adding "
+                                                    <> "a version number to the extension of the chunk file."
+                                                )
+                                        )
```

Comment:
I think this option should be removed in-line with the comment I have above. That would also make it in-line with the issue description.